src/google_civic/core.clj

```diff
@@ -1,39 +1,106 @@
 (ns google-civic.core
   (:require [clj-http.client :as client]
             [cheshire.core :as json]
-            [slingshot.slingshot :refer [try+]]))
+            [clojure.set :as set]))
 
 (def api-root "https://www.googleapis.com/civicinfo/v2")
 
+(def temporary-rate-limit-reasons
+  #{"concurrentLimitExceeded" "rateLimitExceeded"
+    "servingLimitExceeded" "userRateLimitExceeded"})
+
+(defn temporary-rate-limit?
+  "True if the errors in the response only include errors from the
+  temporary-rate-limit-reasons set and the code is 403."
+  [response body]
+  (and (= 403 (:status response))
+       (let [errors (get-in body [:error :errors])
+             reasons (->> errors (map :reason) set)]
+         (and (seq reasons)
+              (set/superset? temporary-rate-limit-reasons
+                             reasons)))))
+
+(defn can-retry?
+  "We can retry if the response code indicates a server error, or
+   if the error is deemed a temporary rate limit."
+  [response body]
+  (cond
+    (client/server-error? response) true
+    (temporary-rate-limit? response body) true
+    :else
+    false))
+
 (defn api-url [endpoint]
   (str api-root endpoint))
 
 (defn api-response [api-key endpoint & [query-params]]
   (let [url (api-url endpoint)]
     (client/get url {:accept :json
```

Comment:
`try-try-again` defaults to retrying because that's its intended purpose. It's simple to set the number of tries to 1: `(try-try-again {:tries 1} #(some-fn arg1 ... argn))`.